README.md

```diff
@@ -177,15 +177,14 @@ Now you can connect to the mysql running in the docker container, also to connec
 2. Create a new Env file with the provided content and fill in the RDS public key value.
    ```env
    DB_NAME=${DB_NAME}
-   MYSQL_DB_USERNAME=testuser
-   MYSQL_DB_PASSWORD=testpassword
-   MYSQL_ROOT_PASSWORD=password
+   MYSQL_USERNAME=testuser
```

Comment:
Any reason why you're changing this back to `MYSQL_USERNAME`? I had changed it to `MYSQL_ROOT_USERNAME` in my previous PR since setup was failing.